docs/LwM2M_Client/OpenThread/Adding_device.md

```diff
@@ -0,0 +1,145 @@
+
+Click start commission, and a message should pop up to inform you about the operation's success. ![Start commission](images/commision.png "Start commission"){:style="float: left;margin-right: 1177px;margin-top: 17px; margin-bottom: 17px;"}
+
+### Device configuration
+
+0. Connect the nRF52840 board to the USB port of your machine.
+
+0. Because NCS uses a different Zephyr version, it is necessary to change our Zephyr workspace, it is handled by using a different manifest file.
+Set West manifest path to *Anjay-zephyr-client/demo*, and manifest file to *west-nrf.yml* and do *west update*.
+
+    ```
+        west config manifest.path Anjay-zephyr-client/demo
+        west config manifest.file west-nrf.yml
+        west update
+    ```
+
+0. Go to *Anjay-zephyr-client/demo/boards* folder and find `nrf52840dk_nrf52840.conf` file. In this file, you will need to change the joiner PSKd password (line 41 in the .conf file). ![Configuration file](images/conf_file.PNG){:style="float: left;margin-right: 1177px;margin-top: 17px; margin-bottom: 17px;"}
+
+    !!! Note
+        The last config option in file `CONFIG_OPENTHREAD_FTD` tells about a Full Thread Device (FTD) which always has its radio on and maintains IPv6 address mappings. This option can be changed to `CONFIG_OPENTHREAD_MTD`, a Minimal Thread Device (MTD) forwards all messages to its Parent.
+
+0. Now in demo directory you can compile the project.
+
+    ```
+        west build -b nrf52840dk_nrf52840
+    ```
+
+0. After successful build you can flash the target.
+
+    ```
+        west flash
+    ```
+
+## Connecting to the LwM2M Server
+
+To connect to Coiote IoT Device Management LwM2M Server, please register at [https://eu.iot.avsystem.cloud](https://eu.iot.avsystem.cloud).
+
+To connect the board:
+
+1. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+2. In **Device Inventory**, click **Add device**.
+3. Select the **Connect your LwM2M device directly via the Management server** tile.
+       ![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+    3. In the **Device credentials** step:
+         - In the **Device ID** enter your board endpoint name, e.g. `test_device`.
+             ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+         - In the **Security mode** section, select the **PSK (Pre-Shared Key)** mode:
+              - In the **Key identity** field, type the same name as in the `Endpoint name` field
+              - In the **Key** field, type the shared secret used in the device-server authentication.
+    4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+    5. In the **Connect your device** step, follow the [next section](#configuring-the-client) to run the client and connect it to the server.
+
+
+## Configuring the Client
+
+0. With the board still connected to a serial port interface, open a serial communication program.
```

Comment:
```suggestion
0. With the board still connected to a serial port interface, open a serial communication program (Zephyr logs should appear on the screen). 
```
or something.

Reply:
It's copy and paste part from the other tutorial. If we want to change this here we should change this everywhere.